### PR TITLE
changing the default csv separator

### PR DESCRIPTION
**Pull request by calmacx**

### CODE REVIEW

File: coconnect/cli/subcommands/map.py [hunk 1]
```diff
@@ -68,6 +68,10 @@ def remove_class(ctx,name):
               default='csv',
               type=click.Choice(['csv']),
               help="specify the type of inputs, the default is .csv inputs")
+@click.option("--csv-separator",
+              default=None,
+              type=click.Choice([';',':','\t',',',' ',]),
```
Comment (calmacx) [inline coconnect/cli/subcommands/map.py:73]: probably unnecessary

File: coconnect/cli/subcommands/map.py [hunk 2]
```diff
@@ -136,6 +140,11 @@ def run(ctx,rules,inputs,
                                         inputs=inputs,
                                         output_folder=output_folder,
                                         use_profiler=use_profiler)
+
+    #allow the csv separator to be changed
+    #the default is tab (\t) separation
+    if not csv_separator is None:
+        cdm.set_csv_separator(csv_separator)
```
Comment (calmacx) [inline coconnect/cli/subcommands/map.py:147]: maybe not allow it to be changed in the simple etltool?

**Comment by calmacx** [comment]
hi @spco 

here's a small PR if you're bored in your meetings. I have a few comments to myself, let me know what you think

cheers
Calum

**Comment by spco** [comment]
I think this looks good - the addition of the CL argument might be overkill, but it has the right default so I'm happy with that. Good to merge from my side.

**Comment by calmacx** [comment]
cheers @spco

**Pull request merged**